src/Tickets/Seating/Admin/Events/Associated_Events.php

```diff
@@ -0,0 +1,282 @@
+<?php
+/**
+ * Associated Events list table.
+ */
+
+namespace TEC\Tickets\Seating\Admin\Events;
+
+if ( ! class_exists( 'WP_List_Table' ) || ! class_exists( 'WP_Posts_List_Table' ) ) {
+	require_once ABSPATH . 'wp-admin/includes/screen.php';
+	require_once ABSPATH . 'wp-admin/includes/class-wp-list-table.php';
+	require_once ABSPATH . 'wp-admin/includes/class-wp-posts-list-table.php';
+}
+
+use TEC\Tickets\Seating\Meta;
+use Tribe__Tickets__Admin__Columns__Tickets;
+use Tribe__Tickets__Main as Tickets;
+use WP_Posts_List_Table;
+use WP_Post;
+
+/**
+ * Associated Events list table.
+ *
+ * @since TBD
+ */
+class Associated_Events extends WP_Posts_List_Table {
+	/**
+	 * @var int Items count.
+	 */
+	private int $item_count = 0;
+	/**
+	 * The constructor.
+	 *
+	 * @since TBD
+	 */
+	public function __construct() {
+		parent::__construct(
+			[
+				'singular' => 'Event',
+				'plural'   => 'Events',
+			]
+		);
+	}
+	
+	/**
+	 * Get the columns.
+	 *
+	 * @since TBD
+	 *
+	 * @return string[]
+	 */
+	public function get_columns() {
+		return [
+			'title'   => __( 'Title', 'event-tickets' ),
+			'tags'    => __( 'Tags', 'event-tickets' ),
+			'tickets' => __( 'Attendees', 'event-tickets' ),
+			'date'    => __( 'Date', 'event-tickets' ),
+		];
+	}
+	
+	/**
+	 * Get the default column.
+	 *
+	 * @since TBD
+	 *
+	 * @param WP_Post $item The current WP_Post object.
+	 * @param string  $column_name The column name.
+	 *
+	 * @return string
+	 */
+	public function column_default( $item, $column_name ) {
+		switch ( $column_name ) {
```

Comment:
This `switch` does not switch and will always return `$item->post_title`, refactor.